homeworks/roman.buzukov_BuzRom/4-modules/js/asterisk.js

```diff
@@ -0,0 +1,7 @@
+export function addHours(date, hours) {
+    return new Date(new Date(date).setHours(date.getHours() + hours)).toLocaleTimeString().split(':').join(' : ');
```

Comment:
No need to make two Date objects. You can create one, setHours for it, and work with it further.

Reply:
Also, this line is too long and difficult to understand. Split it into some simpler lines.